tiledb/ml/models/sklearn.py

```diff
@@ -72,10 +99,30 @@ def _create_array(self):
             schema = tiledb.ArraySchema(
                 domain=dom,
                 sparse=False,
-                attrs=attrs,
+                attrs=attrs, ctx=tiledb_create_context
             )
 
             tiledb.Array.create(self.uri, schema)
+
+            if self.tiledb_uri:
```

Comment:
Maybe this part could be separate update function.

Reply:
A separate function now